proto/accessgraph/v1alpha/access_graph_service.proto

```diff
@@ -82,6 +83,9 @@ service AccessGraphService {
 
   // GitlabEventsStream is a stream of commands to the Gitlab importer.
   rpc GitlabEventsStream(stream GitlabEventsStreamRequest) returns (stream GitlabEventsStreamResponse);
+
+  // EntraEventsStream is a stream of commands to the Entra ID SSO importer.
+  rpc EntraEventsStream(stream EntraEventsStreamRequest) returns (stream EntraEventsStreamResponse);
```

Comment:
It's used to send Entra events, but shouldn't be called Azure as the data comes from Azure and we will probably use it to send more Azure related things in the future?

Reply:
The issue here is that this is supposed to be called from Auth server while the other can be from discovery service tho

Reply:
Yeah, this is basically called `Entra` because Azure resources will likely be synced by a different loop in discovery, and thus would use a different stream.

Both processes will be using the `azure-oidc` integration, but the plugin will only be responsible for the Entra directory part (users, groups, applications, ...)